build-system/compile/sources.js

```diff
@@ -39,11 +39,17 @@ const COMMON_GLOBS = [
   'third_party/timeagojs/**/*.js',
   'third_party/vega/**/*.js',
   'third_party/webcomponentsjs/ShadowCSS.js',
+  'node_modules/dompurify/package.json',
   'node_modules/dompurify/dist/purify.es.js',
+  'node_modules/promise-pjs/package.json',
   'node_modules/promise-pjs/promise.mjs',
+  'node_modules/web-animations-js/package.json',
   'node_modules/web-animations-js/web-animations.install.js',
+  'node_modules/web-activities/package.json',
   'node_modules/web-activities/activity-ports.js',
+  'node_modules/@ampproject/animations/package.json',
   'node_modules/@ampproject/animations/dist/animations.mjs',
+  'node_modules/@ampproject/worker-dom/package.json',
```

Comment:
Why is it insufficient to include just the lines ending in `package.json`? Why do we still need the old ones?

Reply:
Closure requires both. Adding a line here means closure will use the file during compilation, so we need every file that closure should read.